dev-ops/common/actions/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#DESCRIPTION: Update Shopware after deployment
+
+INCLUDE: ./cache.sh
+
+bin/console database:refresh-migration
```

Comment:
This command requires a path to a migration and refreshes  the timestamp. It should not be necessary.

Reply:
OK, I'll remove it